ui: change 'Physcial Replication Producer' category to 'Replication Producer'

msbutler opened the pull request:
Epic: none

Release note: none

msbutler committed:
ui: change 'Physcial Replication Producer' category to 'Replication Producer'

Epic: none

Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/127374)
<!-- Reviewable:end -->

msbutler committed:
ui: add Logical Replication Ingestion Category

Epic: none

Release note: none

msbutler committed:
ui: only show LDR badge on jobs page

The patch avoids showing the fraction completed on the jobs page for LDR
because it doesn't mean anything.

Epic: none

Release note: none

stevendanna reviewed (approved):
LGTM!

msbutler commented:
TFTR!

bors r=stevendanna

msbutler commented:
bors r-

msbutler commented:
bors r+
